pilot/pkg/bootstrap/servicecontroller.go

```diff
@@ -75,6 +76,7 @@ func (s *Server) initKubeRegistry(args *PilotArgs) (err error) {
 	args.RegistryOptions.KubeOptions.MeshWatcher = s.environment.Watcher
 	args.RegistryOptions.KubeOptions.SystemNamespace = args.Namespace
 	args.RegistryOptions.KubeOptions.MeshServiceController = s.ServiceController()
+	args.RegistryOptions.KubeOptions.SyncTimeout = features.RemoteClusterTimeout
```

Comment:
I don't think we want the timeout behavior for the "local" cluster.

Reply:
Reasonable, done